R/utils.R

```diff
@@ -1,3 +1,12 @@
 generate_id <- function() {
   paste0(sample(letters, 26), collapse = "")
-}
\ No newline at end of file
+}
+
+prep_element <- function (el) if (!grepl("^\\.", el) && !grepl("^\\#", el)) paste0("#", el) else el
```

Comment:
We don't check for all valid selectors, only classes and ids.

See https://github.com/JohnCoene/cicerone/pull/36#pullrequestreview-859792163

The following are valid CSS selectors but this function would prepend `#` to them.

```r
prep_element("#id")
prep_element(".class")
prep_element("parent child")
prep_element(".parent #child")
prep_element("parent > child")
prep_element("[data-thing='something']")
```